service/src/main/resources/db/changesets/20230530_avoid_postgres_specific_features.yaml

```diff
@@ -1,7 +1,23 @@
 databaseChangeLog:
+  - property:
+      dbms: postgresql
+      name: _id.type
```

Comment:
Why are the names prefixed with an underscore in this changeset file and not in the previous one? I think it would be easier to follow if they are the same in both places.

Reply:
no reason, not sure if property names must be unique across changelogs. I will remove the `_` and check

Reply:
maybe there is also a global properties file... I will check.